config/options.go

```diff
@@ -1248,19 +1248,20 @@ func (o Options) indexCerts(ctx context.Context) certsIndex {
 			idx.addCert(cert)
 		}
 	} else if o.Cert != "" {
-		if data, err := base64.StdEncoding.DecodeString(o.Cert); err != nil {
-			log.Error(ctx).Err(err).Msg("bad base64 for local cert: skipped")
-		} else if cert, err := cryptutil.ParsePEMCertificate(data); err != nil {
+		if cert, err := cryptutil.ParsePEMCertificateFromBase64(o.Cert); err != nil {
 			log.Error(ctx).Err(err).Msg("parsing local cert: skipped")
 		} else {
 			idx.addCert(cert)
 		}
 	}
 
 	for _, c := range o.CertificateFiles {
-		cert, err := cryptutil.ParsePEMCertificateFromFile(c.CertFile)
+		cert, err := cryptutil.ParsePEMCertificateFromBase64(c.CertFile)
```

Comment:
Why a B64-encoded cert may end up in `CertFile`?

Reply:
The current settings store raw certificates as a base64 encoded string. I either don't know why or don't remember why. Changing this setting would be a breaking change.

Reply:
We do not have separate keys for this: https://www.pomerium.com/docs/reference/certificates

```yaml
certificates:
  - cert: "$HOME/.acme.sh/authenticate.example.com_ecc/fullchain.cer"
    key: "$HOME/.acme.sh/authenticate.example.com_ecc/authenticate.example.com.key"
  - cert: "$HOME/.acme.sh/verify.example.com_ecc/fullchain.cer"
    key: "$HOME/.acme.sh/verify.example.com_ecc/verify.example.com.key"
  - cert: "$HOME/.acme.sh/prometheus.example.com_ecc/fullchain.cer"
    key: "$HOME/.acme.sh/prometheus.example.com_ecc/prometheus.example.com.key"
```

Reply:
oh I see we accept both files and b64 strings there. 

may I suggest we resolve the confusion by 
- renaming `CertificateFiles` var into `Certificates` 
- renaming `certificateFilePair` type into `certificatePair`
- adding `certBytes` and `keyBytes` to `certificatePair`
- writing a custom mapstructure func to parse the `certificates` field and decide there if it's a b64 string or a file path, and fill `certFile`/`certBytes` and `keyFile`/`keyBytes` accordingly. As I understand, it may only happen while parsing a config file, and is not the case for converting Protobuf `Settings` into `Options`, as the former only has `CertBytes`/`KeyBytes` fields.

Reply:
Can we pull the above into a separate issue / proposal because -- unless I am missing something -- this would be a breaking change?

Reply:
Sure, we can eject refactoring into separate issue. This PR fixes the original issue with spillover of the cert data into logs. 

NB: what I suggested above would not constitute a breaking change.